flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+    "only-arches": [
+        "x86_64"
```

Comment:
this is disappointing.

Reply:
While compiling avalonia for ARM is possible, some binaries that are needed to make this IDE work fully are not available (yet). Also I would prefer to test this before allowing arm builds